test/message.js

```diff
@@ -84,6 +84,10 @@ const main = async () => {
   for await (const dirent of dir) {
     const ext = extname(dirent.name)
     if (ext === '.js' || ext === '.mjs') {
+      if (typeof AbortSignal === 'undefined' && dirent.name.startsWith('test_runner_abort')) {
+        console.log('no AbortSignal support, skipping', dirent.name)
```

Comment:
would the smoothest flow be to include an AbortSignal ponyfill for node 14? Are there any major concerns including one?

Reply:
I think this would be the nicest option, as long as node 14 is in maintenance lts

Reply:
I couldn't find any satisfying ponyfill. Note that we are running tests both with and without `--experimental-abort-controller` flag, folks who want/need that functionality should pass this flag.

Reply:
right, I think this is a good compromise 👍

Reply:
+1